src/containers/HomeScreenContainer.js

```diff
@@ -1,18 +1,40 @@
 import {connect} from 'react-redux';
 import Home from '../screens/Home';
-import {setCategories, setCurrentCategory, setPhrases} from '../redux/actions';
-import {categoriesRoot, nativeLanguageRoot} from '../redux/selectors';
+import {
+  setCategories,
+  setCurrentCategory,
+  setPhrases,
+  setLearntPhrases,
+  synchronizeStorageToRedux,
+  setSeenPhrases,
+  addSeenPhrase,
+} from '../redux/actions';
+
+import {
+  categoriesRoot,
+  nativeLanguageRoot,
+  learntPhrases,
+  seenPhrases,
+  leftPhrases,
+} from '../redux/selectors';
 
 function mapStateToProps(state) {
   return {
     categories: categoriesRoot(state),
     nativeLanguage: nativeLanguageRoot(state),
+    learntPhrases: learntPhrases(state),
+    seenPhrases: seenPhrases(state),
+    leftPhrases: leftPhrases(state),
   };
 }
 const mapDispatchToProps = {
   setCategories,
   setCurrentCategory,
   setPhrases,
+  setLearntPhrases,
+  synchronizeStorageToRedux,
+  setSeenPhrases,
+  addSeenPhrase,
```

Comment:
It seems like the Home screen does not need/ use many of these actions, for example addSeenPhrase. Which can be removed?